templates/zerver/help/public-communities-directory.md

```diff
@@ -0,0 +1,36 @@
+# Communities directory
+
+{!communities-directory-intro.md!}
+
+To be eligible, an organization must have at least one [public access
+option](/help/web-public-streams) or allow users to [join without an
+invitation][join-without-invite].
+
+## Opt in to Zulip's communities directory
+
+{!admin-only.md!}
+
+{start_tabs}
+
+{settings_tab|organization-profile}
+
+1. Under **Organization profile**, toggle **List organization on
+   zulip.com**.
+
+{!save-changes.md!}
+
+{end_tabs}
+
+!!! warn ""
+
+    Opting in does not guarantee that an organization will be listed
+    on Zulip's communities directory.
```

Comment:
I like the idea of adjusting this phrase to explain that "it gives Zulip permission to list you" or something.